CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 1.3.0
+### Login SSO Documentation
+
+-  Fix a bug in the Login SSO sample code
+
+### Add pagination to `useSearch`
+
+- Update `README.me`
+- Add a new param to the `useSearch` hook: `page`
+- Get the results based on the provided `page`.
+
+*Resolves [#38](https://github.com/bigcommerce/storefront-data-hooks/issues/38)*
+
+### Return `basePrice` in the product query
+
+- Return `basePrice` in the product query
+- Update GraphQL schema to generate the updated types (also includes latest bigcommerce changes)
```

Comment:
What are the latest changes that you are referring to here? Again this is just something that being a bit more specific on will help external devs and us in referencing this historically :)

Reply:
The Graphql schema is generated based on the [Storefront GraphQL API](https://developer.bigcommerce.com/changelog#labels/storefront-api), so we have some small changes from the last updates.

I have clarified the comment.